README.md

```diff
@@ -444,6 +445,51 @@ Medium.
 - [Idea discussion](https://rust-lang.zulipchat.com/#narrow/channel/421156-gsoc/topic/Idea.3A.20Extend.20testing.20of.20.60std.3A.3Aarch.60.20intrinsics)
 - [t-libs/stdarch](https://rust-lang.zulipchat.com/#narrow/channel/208962-t-libs.2Fstdarch)
 
+### Add safety contracts
+
+**Description**
+
+We have taken a goal to
+[instrument the Rust standard library with safety contracts](https://rust-lang.github.io/rust-project-goals/2025h1/std-contracts.html).
+With this approach we are moving from informal comments specifying safety
+requirements on `unsafe` functions to executable Rust code.
+To prioritize which functions to equip with contracts first, we run a
+verification contest in the
+[verify-rust-std](https://github.com/model-checking/verify-rust-std) fork.
+In this contest,
+[challenges](https://model-checking.github.io/verify-rust-std/challenges.html)
+have been put up that request specific sets of functions to be equipped with
+contracts.
+
+Among others, we are currently looking for contributions towards the following challenges:
```

Comment:
I think we should mention that the contributor could also propose their own set of functions to tackle.

We could also include porting contracts to Rust main repo. It's up to you.